apis/binding/v1alpha1/servicebinding_types.go

```diff
@@ -28,73 +28,74 @@ var templates = map[string]string{
 	"lowercase": "{{ .name | lower }}",
 }
 
-// ServiceBindingSpec defines the desired state of ServiceBinding
+// ServiceBindingSpec defines the desired state of ServiceBinding.
 type ServiceBindingSpec struct {
-	// MountPath is the path inside app container where bindings will be mounted
-	// If `SERVICE_BINDING_ROOT` env var is present, mountPath is ignored.
-	// If `SERVICE_BINDING_ROOT` is absent and mountPath is present, set `SERVICE_BINDING_ROOT` as mountPath value
-	// If `SERVICE_BINDING_ROOT` is absent but mounthPath is absent, set   SERVICE_BINDING_ROOT as `/bindings`
-	// When mountPath is used, the file will be mounted directly under that directory
-	// Otherwise it will be under `SERVICE_BINDING_ROOT`/<SERVICE-BINDING-NAME>
+	// Specifies the path inside the app container where bindings will be
+	// mounted.  Setting the environment variable `SERVICE_BINDING_ROOT` will
+	// cause this field to be ignored.  If neither of these are set, this field
+	// defaults to `/bindings`.  This results in the file being mounted under
+	// the directory specified.
 	// +optional
 	MountPath string `json:"mountPath,omitempty"`
 
-	// NamingStrategy defines custom string template for preparing binding names.
-	// It can be pre-defined strategies(i.e none,uppercase), in case strategy provided in this field isn't defined
-	// we are going to treat the value as a custom template and prepare binding names accordingly.
+	// Defines custom string template for preparing binding names.  It can be
+	// set to pre-defined strategies: `none`, `lowercase`, or `uppercase`.
+	// Otherwise, it is treated as a custom template, and it is handled
+	// accordingly.
```

Comment:
golang documentation needs to start with field name: https://go.dev/blog/godoc

Reply:
Fixed.